### PR TITLE
[hi] Localize content/hi/docs/reference/glossary/cri.md

### PR DESCRIPTION
**Pull request by amolmote**
Signed-off-by: amolmote <amolmote201@gmail.com>
Fixex #35600  Localize content/hi/docs/reference/glossary/cri.md

### CODE REVIEW

File: content/hi/docs/reference/glossary/cri.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+
+aka:
+tags:
+- fundamental
+---
+कंटेनर रनटाइम इंटरफ़ेस (सीआरआई) कंटेनर रनटाइम के लिए एक एपीआई है जो नोड पर क्यूबलेट के साथ एकीकृत होता है।
+<!--more-->
+
+अधिक जानकारी के लिए, [सीआरआई](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-node/container-runtime-interface.md) एपीआई और विनिर्देश देखें।
```
Comment (amolmote) [inline content/hi/docs/reference/glossary/cri.md:17]:
```suggestion
अधिक जानकारी के लिए, [CRI](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-node/container-runtime-interface.md) API और विनिर्देश देखें।
```

File: content/hi/docs/reference/glossary/cri.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+title: कंटेनर रनटाइम इंटरफ़ेस (Container runtime interface)
+id: cri
+date: 2019-03-07
+full_link: /docs/concepts/overview/components/#container-runtime
+short_description: >
+    क्यूबलेट के साथ एकीकृत करने के लिए कंटेनर रनटाइम के लिए एक एपीआई
```
Comment (amolmote) [inline content/hi/docs/reference/glossary/cri.md:7]:
```suggestion
    क्यूबलेट के साथ एकीकृत करने के लिए कंटेनर रनटाइम के लिए एक API
```

**Review by amolmote — COMMENTED** [review]
I think I have made those mistakes. other reviewer also see this. should I commit this changes?

**Review by divya-mohan0209 — COMMENTED** [review]
/lgtm

**Review by anubha-v-ardhan — COMMENTED** [review]
/assign
/approve

Thank you @amolmote :)

**Pull request merged**